docs/pages/reference/access-controls/roles.mdx

```diff
@@ -142,6 +144,55 @@ spec:
       'workload': ['database', 'backup']
 ```
 
+### `client_idle_timeout`
+
+This example configures a global idle timeout of **30 minutes**. After 30 minutes of inactivity, the session will be terminated.
+
+The `client_idle_timeout` in Teleport is a configurable setting that helps improve security by terminating inactive sessions after a specified period. It can be applied globally or per role, allowing for flexibility based on your organization's security policies.
+   The `client_idle_timeout` configuration ensures that SSH sessions, web application, kubectl, or database connection that remain inactive for a certain period of time
+   are automatically terminated. You can also use it to set limits for SSH sessions initiated *from* the web interface (note that this is distinct from `web_idle_timeout`, which sets idle timeout for the Web UI). 
+   This helps to mitigate risks associated with unattended sessions, such as unauthorized access.
+
+#### How it works
+   Teleport monitors user activity within sessions (like key presses or mouse movement).
+   If there is no detected activity for the duration defined by `client_idle_timeout`, the session is terminated, forcing the user to reconnect.
+   
+#### Configuration
+   The `client_idle_timeout` can be configured globally or per role, giving administrators flexibility in how they apply idle timeout rules.
+
+   - **Global configuration** (applies to all users):
+
+You can set the `client_idle_timeout` globally in the Teleport cluster configuration (`teleport.yaml`) under the `auth_service` section:
+
+ 	```yaml
+ 	auth_service:
+   	client_idle_timeout: 15m
```

Comment:
```suggestion
   	  client_idle_timeout: 15m
```

`client_idle_timeout` goes under the `auth_service` section, so it needs to be indented.